src/hex_core.hrl

```diff
@@ -1 +1,7 @@
 -define(HEX_CORE_VERSION, "0.7.0").
+
+-ifdef(fun_stacktrace).
```

Comment:
You could go with `-ifdef(OTP_RELEASE).` here and prevent the change in `rebar.config`.